graspy/inference/latent_distribution_test.py

```diff
@@ -125,37 +153,75 @@ def __init__(
         if not isinstance(n_bootstraps, int):
             msg = "n_bootstraps must be an int, not {}".format(type(n_bootstraps))
             raise TypeError(msg)
-        elif n_bootstraps < 1:
-            msg = "{} is invalid number of bootstraps, must be greater than 1"
+        elif n_bootstraps < 0:
+            msg = "{} is invalid number of bootstraps, must be positive"
             raise ValueError(msg.format(n_bootstraps))
 
         if not isinstance(workers, int):
             msg = "workers must be an int, not {}".format(type(workers))
             raise TypeError(msg)
-        elif workers <= 0:
-            msg = "{} is invalid number of workers, must be greater than 0"
+        elif workers == 0 or workers < 1:
```

Comment:
how does this allow workers to be -1? also, don't some systems allow -2 for all cores but one? not sure what the backend is here ultimately but sometimes that's a thing

Reply:
it doesn't.

this is me being sloppy. meant to be
`elif workers == 0 or workers < -1`

thanks for the catch.

Reply:
as far as i can tell this ultimately lands [here](https://github.com/neurodata/hyppo/blob/06ef0fc0bc450b60cf8cab07f1923e3df0d613d9/hyppo/_utils.py#L139)
if true, joblib will take ints less than -1, e.g. -2 means use all cores but one (which I use often). hyppo never even checks this value. I feel like maybe we just make sure it's an int for now?

Reply:
fine with me.

Reply:
resolved.